### PR TITLE
Add support for tags

### PR DESCRIPTION
**Pull request by nfcampos**
Fixes # (issue)

#### Who can review?

Tag maintainers/contributors who might be interested:

### CODE REVIEW

File: langchain/callbacks/manager.py [hunk 1]
```diff
@@ -425,10 +436,13 @@ async def on_llm_error(
 class CallbackManagerForChainRun(RunManager, ChainManagerMixin):
     """Callback manager for chain run."""
 
-    def get_child(self) -> CallbackManager:
+    def get_child(self, tag: str | None = None) -> CallbackManager:
```
Comment (vowelparrot) [inline langchain/callbacks/manager.py:439]: It doesn't impact anything python and not w/in a dataclass but this syntax isn't really supported in py 3.8

**Review by vowelparrot — APPROVED** [review]

File: langchain/callbacks/manager.py (hunk 1 above)
Comment (nfcampos) [reply langchain/callbacks/manager.py]: Cool, fixed

File: langchain/callbacks/base.py
Comment (agola11) [inline langchain/callbacks/base.py:1]: The tags parameter needs to be present in the `CallbackManagerMixin` methods too, no? Currently they are only in the async methods.

**Comment by joebordes** [comment]
where is this documented? what is the functionality?

**Comment by nfcampos** [comment]
@joebordes see here https://github.com/hwchase17/langchain/pull/6155/files

**Comment by joebordes** [comment]
thank!

**Comment by AkshayVerma26** [comment]
```
File "/Users/akshayverma/langchain/langchain/callbacks/tracers/langchain.py", line 11, in <module>
    from langchainplus_sdk import LangChainPlusClient
ModuleNotFoundError: No module named 'langchainplus_sdk'
```
How to resolve this error?

**Pull request merged**